lib/tasks/db.rake

```diff
@@ -0,0 +1,46 @@
+namespace :db do
+
+  desc 'Dumps the database to file'
+  task :dump => :environment do
+
+    host = ActiveRecord::Base.connection_config[:host]
+    db = ActiveRecord::Base.connection_config[:database]
+
+    cmd = "pg_dump --no-owner --no-acl -v -c -f #{dump_file} --no-password "
+    cmd += ENV['DATABASE_URL'] || "--host #{host} --dbname #{db}"
```

Comment:
Ahh OK, that makes sense. Perhaps a rake command explicitly for testing might be a better way of splitting that out :)